src/components/modules/dragNDrop.ts

```diff
@@ -89,28 +112,104 @@ export default class DragNDrop extends Module {
      * Try to set current block by drop target.
      * If drop target is not part of the Block, set last Block as current.
      */
-    const targetBlock = BlockManager.setCurrentBlockByChildNode(dropEvent.target as Node);
+    let targetBlock = BlockManager.setCurrentBlockByChildNode(dropEvent.target as Node);
 
     if (targetBlock) {
-      this.Editor.Caret.setToBlock(targetBlock, Caret.positions.END);
+      if (targetBlock.dropTarget === BlockDropZonePosition.Top) {
```

Comment:
I suggest to rename this property, as it means a different thing now. Maybe `dragState`?

Reply:
It describes the position where we can drop dragging blocs. Not the dragging block itself. Maybe `dropZonePosition`?